docs/source/index.rst

```diff
@@ -32,6 +32,7 @@ Pyro Documentation
    contrib.autoname
    contrib.bnn
    contrib.easyguide
+   contrib.examples
```

Comment:
Do you intend for `contrib.examples` to house utilities for examples? Maybe we could also just move our examples module here too (not in this PR).

Reply:
I think it's ok for examples to live outside of the repo, and nice for some data downloaders to live in the repo. Often R packages are organized that way, I believe.

Reply:
Oh..I didn't mean the examples directory. I prefer that being outside the main package. I thought we had a separate `pyro.examples` module, but I was mistaken - it was already in `pyro.contrib`.